cscs-checks/microbenchmarks/kernel_latency/kernel_latency.py

```diff
@@ -2,14 +2,12 @@
 import reframe.utility.sanity as sn
 
 
-@rfm.required_version('>=2.16-dev0')
```

Comment:
Why did you remove this? It's still valid, since you're using units in the reference tuples.